kroxylicious-operator/install/03.Deployment.kroxylicious-operator.yaml

```diff
@@ -33,7 +33,18 @@ spec:
           resources:
             limits:
               memory: 300M
-              cpu: 1
+              cpu: "1"
             requests:
               memory: 300M
-              cpu: 1
+              cpu: "1"
+          ports:
+            - containerPort: 8080
+              name: http
+          livenessProbe:
+            initialDelaySeconds: 10
+            periodSeconds: 10
```

Comment:
Is there some benefit to having a higher period with lower failure threshold vs retrying more frequently with a higher threshold?

Reply:
Honestly, I just made these up without thinking very hard. The only data point I have is it looks like it takes the operator between 5s and 10s to get to the point of having started on my machine.